pkg/console/controllers/healthcheck/controller.go

```diff
@@ -99,13 +113,20 @@ func (c *HealthCheckController) Sync(ctx context.Context, controllerContext fact
 	default:
 		return fmt.Errorf("unknown state: %v", updatedOperatorConfig.Spec.ManagementState)
 	}
-
-	statusHandler := status.NewStatusHandler(c.operatorClient)
-
 	ingressConfig, err := c.ingressClient.Get(ctx, api.ConfigResourceName, metav1.GetOptions{})
 	if err != nil {
+		klog.Errorf("ingress config error: %v", err)
 		return statusHandler.FlushAndReturn(err)
 	}
+	infrastructureConfig, err := c.infrastructureClient.Get(ctx, api.ConfigResourceName, metav1.GetOptions{})
+	if err != nil {
+		klog.Errorf("infrastructure config error: %v", err)
+		return statusHandler.FlushAndReturn(err)
+	}
+	// disable the health check for internal NLB
```

Comment:
Can we please add a code comment on why we disable it or include a link to https://issues.redhat.com/browse/OCPBUGS-23300?

e.g
// Disable the health check for control plane external topology (hypershift) and ingress NLB.
// This is to avoid an issue with internal NLB see https://issues.redhat.com/browse/OCPBUGS-23300